desktop/appcontainer/SuspendPanel.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {hoistCmp, XH} from '@xh/hoist/core';
+import {viewport, div, p, filler} from '@xh/hoist/cmp/layout';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {Icon} from '@xh/hoist/icon';
+
+import './SuspendPanel.scss';
+
+/**
+ * Generic Panel to display when the app is suspended.
+ * @private
+ */
+export const suspendPanel = hoistCmp.factory({
+    displayName: 'SuspendPanel',
+
+    render() {
+        const message = XH.suspendData?.message ?? 'Your application requires a restart.';
```

Comment:
There is no API for setting the suspendData.message property - should we make the WebsocketsPanel in the admin console more like the test message (where the message is defaulted in the prompt)?

Reply:
So I had that message teed up for when we suspend based on a min version.  I was thinking it was overkill to tie it to the socket based kill for now.....  Realize I probably could have left the message out until we needed it, but also wanted to make it clear how I saw the need for a `suspendData` rather than just a `suspendReason`

Reply:
Great catch on the extra state